game/state/gameevent.cpp

```diff
@@ -124,9 +124,9 @@ UString GameVehicleEvent::message()
 				return tr("An illegal flyer has been detected.");
 			}
 		case GameEventType::NotEnoughAmmo:
-			return tr("Not enough ammo to rearm vehicle");
+			return format("%s %s", tr("Not enough ammo to rearm vehicle:"), vehicle->name);
```

Comment:
This should probably have the "%s" placeholder thingie in the tr() string - just in case other languages don't make sense having it at the end

Reply:
I am not sure what you mean by this. Maybe you can change this one to fit other languages?